models/faculty/faculty.js

```diff
@@ -0,0 +1,100 @@
+import connector from "#models/databaseUtil";
+import { subjectSchema } from "#models/subjects";
+import { departmentSchema } from "#models/department";
```

Comment:
if youre using ref in your code instead of using subdocuments, theres no need to import these.